site2/docs/functions-package-java.md

```diff
@@ -10,7 +10,23 @@ For the runtime Java version, refer to [Pulsar Runtime Java Version Recommendati
 
 :::
 
-To package a Java function, complete the following steps.
+There are two methods to package Java Functions, that is [uber JAR](#package-as-jar) and [NAR](#package-as-nar).
+
+:::note
+
+If you plan to package and distribute your function for others to use, you are obligated to
+license and copyright your own code properly. Remember to add the license and copyright to
+all libraries your code uses and to your distribution.
+
+:::
+
+> If you use the [NAR](#package-as-nar) method, the NAR plugin
+automatically creates a `DEPENDENCIES` file in the generated NAR package, including the proper
+licensing and copyrights of all libraries of your function.
```

Comment:
They seem similar notes/tips, why different syntax?

You may read the syntax guide (https://pulsar.apache.org/contribute/document-syntax/#admonitions) for what is expected here.

Reply:
done